lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -282,15 +281,21 @@ bool LayerTreeItem::isLocked() const { return _layer && _layer->PermissionToEdit
 
 bool LayerTreeItem::appearsLocked() const
 {
-    if (isLocked()) {
-        return true;
+    auto item = parentLayerItem();
+    if (item != nullptr) {
+        return item->isLocked();
     }
+
+    return false;
+}
+
+bool LayerTreeItem::isSystemLocked() const { return MayaUsd::isLayerSystemLocked(_layer); }
+
+bool LayerTreeItem::appearsSystemLocked() const
```

Comment:
Named similar to appearsLocked(). It has to do with the parent / children's state of system lock.
By design, some of the sublayer's actions cannot be performed if the parent is locked.

Reply:
Oh, so you want to be able to distinguish between something being locked and something being locked by its parent? So that is why appearsLocked does not check itself? OK, but the function name is a bit misleading. You can keep them like that, just a trap for the unwary :)

Reply:
This is similarly named to appearsMuted. The notion is that something "appears" locked but it's not really locked itself. Hence the word appear.